cvat-core/src/log.ts

```diff
@@ -137,74 +148,45 @@ class LogWithObjectsInfo extends Log {
     }
 }
 
-class LogWithWorkingTime extends Log {
-    public validatePayload(): void {
-        super.validatePayload.call(this);
-
-        if (
-            !('working_time' in this.payload) ||
-            !(typeof this.payload.working_time === 'number') ||
-            this.payload.working_time < 0
-        ) {
-            const message = `
-                The field "working_time" is required for ${this.type} log. It must be a number not less than 0
-            `;
-            throw new ArgumentError(message);
-        }
-    }
-}
-
 class LogWithExceptionInfo extends Log {
```

Comment:
Do we really need a separate class? I believe it can be decided based on LogType (maybe EventType) value.